app/controllers/laa_references_controller.rb

```diff
@@ -22,8 +22,23 @@ def new; end
   def create
     authorize! :create, :link_maat_reference, message: I18n.t('unauthorized.default')
 
-    link_laa_reference_and_redirect && return if @link_attempt.valid?
-    render 'new'
+    @link_attempt.validate!
```

Comment:
I recognise that ` @link_attempt` is a pretty bad name for a form 🤔 I'd be happy to rename it.